cmd/main.go

```diff
@@ -1,34 +1,34 @@
 package main
 
 import (
+	"github.com/CDCgov/data-exchange-csv/cmd/internal/cli"
 	"github.com/CDCgov/data-exchange-csv/cmd/internal/constants"
-	"github.com/CDCgov/data-exchange-csv/cmd/internal/models"
 	"github.com/CDCgov/data-exchange-csv/cmd/internal/processor"
+	"github.com/CDCgov/data-exchange-csv/cmd/internal/utils"
 	"github.com/CDCgov/data-exchange-csv/cmd/internal/validate/file"
-	"github.com/CDCgov/data-exchange-csv/cmd/internal/validate/row"
 	"github.com/CDCgov/data-exchange-csv/cmd/pkg/sloger"
 )
 
 func main() {
-	logger := sloger.With(constants.PACKAGE, constants.MAIN)
-	logger.Info(constants.APPLICATION_STARTED)
 
-	event := "data/event_source.json"
+	validationInputParams := cli.ParseFlags()
 
-	fileValidationResult := file.Validate(event)
-	processor.ProcessFileValidationResult(fileValidationResult)
+	logToAFile := validationInputParams.LogToFile
```

Comment:
Recommend consistent naming - `logToAFile` could probably match the property on `validationInputParams` and be called `logToFile`.